sql/parser/builtins.go

```diff
@@ -412,6 +412,24 @@ var builtins = map[string][]builtin{
 		},
 	},
 
+	"greatest": {
+		builtin{
+			types: nil,
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
+				return pickFromTuple(true /* greatest */, args)
+			},
+		},
+	},
+
+	"least": {
+		builtin{
+			types: nil,
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
+				return pickFromTuple(false /* greatest */, args)
```

Comment:
I prefer `/* !greatest */` for comments like this, to make it clear that we are passing false as the "greatest" parameter instead of suggesting that the parameter is named "least" and we are passing false to it.